frontend/src/components/pages/CampOverview/ViewCamperModal/PickupDropoffTableRow.tsx

```diff
@@ -54,7 +54,7 @@ const PickupDropoffTime = ({
   return (
     <Td pl={0} pt={2} pb={0} color={color}>
       <HStack>
-        {pickup ? <SunriseIcon fill={color} /> : <SunsetIcon fill={color} />}
```

Comment:
we need to revert this, the pickup icon is supposed to be the sunrise icon not the sunset one (I know it's kinda confusing)

Reply:
Expected look:

![Screen Shot 2023-01-28 at 2 12 05 PM](https://user-images.githubusercontent.com/46568041/215286593-e85a6025-2a90-4ac3-8dbd-c56bff0e3a98.png)